whois-query/src/main/java/net/ripe/db/whois/query/hazelcast/HazelcastInstanceManager.java

```diff
@@ -60,7 +62,11 @@ private Config getGenericConfig() {
                 .setProperty("hazelcast.phone.home.enabled", "false")
                 .setProperty("hazelcast.memcache.enabled","false")
                 .setProperty("hazelcast.redo.giveup.threshold","10")
-                .setProperty("hazelcast.logging.type","slf4j");
+                .setProperty("hazelcast.logging.type","slf4j")
+                .setProperty("hazelcast.shutdownhook.enabled","false")
+                .setProperty("hazelcast.graceful.shutdown.max.wait","60");
+
+        config.setCPSubsystemConfig(new CPSubsystemConfig().setPersistenceEnabled(false));
```

Comment:
You could also use `config.getCPSubsystemConfig().setPersistenceEnabled(false);`